### ISSUE TITLE
30$: report misleading indentation for braceless if/etc.

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1113]
```javascript
if (x)
  if (y)
    z();
else
  w();
```
quick-lint-js should warn that the `else` should be indented.

**Comment by vegerot** [comment]
A special case can be

```js
if (x)
  y();
  z();
```

quick-lint should warn that `z()` is indented but is not part of the `if`
